cfme/automate/__init__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from navmazing import NavigateToSibling
+from widgetastic.widget import View
+from widgetastic_patternfly import Accordion, Dropdown
+
+from cfme import BaseLoggedInPage
+from utils.appliance import IPAppliance
+from utils.appliance.endpoints.ui import navigator, CFMENavigateStep
+
+
+class AutomateCustomizationView(BaseLoggedInPage):
+    @property
+    def is_displayed(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Automate', 'Customization'])
+
+    @View.nested
+    class provisioning_dialogs(Accordion):  # noqa
+        ACCORDION_NAME = 'Provisioning Dialogs'
+        CONTAINS_TREE = True
+
+    @View.nested
+    class service_dialogs(Accordion):  # noqa
+        ACCORDION_NAME = 'Service Dialogs'
+        CONTAINS_TREE = True
+
+    @View.nested
+    class buttons(Accordion):  # noqa
+        CONTAINS_TREE = True
+
+    @View.nested
+    class import_export(Accordion):  # noqa
+        ACCORDION_NAME = 'Import/Export'
+        CONTAINS_TREE = True
+
```

Comment:
So far all these views do not contain the "right" side except the forms themselves. I will try figuring out the best way how to specify the summary (InfoBlock) stuff in there. Suggestions welcome :)